### PR TITLE
Feature/39 add no-default-export rule

### LINKED ISSUE #39: import/no-default-export
PR #63 

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md

We should turn on that rule. In some repos (`presense-frontend`, `design-system`), we could easily adapt that rule. In `insights` and `adverity-components`, it is rather difficult. We have 3 options now:

- enable those rules in the repos directly that can already support it (not that good).
- enable it here per default, release as a breaking change, and disable it in repos that still have default exports (better, but breaking).
- enable it here per default as a `warning`, and lint the repos where we already have no default exports with `--max-warnings=0`, which basically makes warning to errors

@namjul @vlabs @danielbartsch 

See also: https://github.com/pmedianetwork/handbook/issues/13

### PR DESCRIPTION
**Pull request by frycz**
Closes #39

### CODE REVIEW

File: .changeset/39-add-no-default-export-rule.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+issue: https://github.com/pmedianetwork/eslint-config/issues/39
+type: minor
```
Comment (TkDodo) [inline .changeset/39-add-no-default-export-rule.md:3]: Not sure if we have reached a consensus yet on the issue on how to implement this rule. But minor release with the rule set to error was definitely NOT on the list.

**Review by TkDodo — CHANGES_REQUESTED** [review]

File: .changeset/39-add-no-default-export-rule.md (hunk 1 above)
Comment (frycz) [reply .changeset/39-add-no-default-export-rule.md:3]: Sorry, my bad.

Comment (TkDodo) [reply .changeset/39-add-no-default-export-rule.md:3]: I added the `discussion` label to the issue.

**Comment by TkDodo** [comment]
let's revisit this later once we have the monorepo.

**Pull request closed without merging**